Library/Formula/cairo.rb

```diff
@@ -23,6 +23,7 @@ class Cairo < Formula
   depends_on "libpng"
   depends_on "pixman"
   depends_on "glib"
+  depends_on :x11 if MacOS::XQuartz.installed?
```

Comment:
I can't imagine we'll re-add X11 support to Cairo, but particularly by default which this essentially does (Our CI have X11, so the bottles generated would be generated with X11 ties). This is actually harsher than our previous `depends_on :x11 => :recommended` dependency as this time there would be no opt-out for anyone with X11 installed.

CC @mikemcquaid @tschoonj